docs/manul_doc/Functions_and_Operators/Operator_Precedence.md

```diff
@@ -0,0 +1,22 @@
+The following list shows the precedence of nGQL operators, in descending order. Operators on a line have the same precedence.
+
+```
+!
+- (unary minus)
+*, /, %
+-, +
+== , >=, >, <=, <, <>, !=
+&&
+||
+= (assignment)
```

Comment:
can assignment be used t

Reply:
only works when assigning "go from" results at this moment, e.g. "$a=GO FROM 201 OVER like; GO FROM $a.id OVER select YIELD $^.student.name;"